File: practice/prime_between_range.c
// PRINT NUMBER IN A GIVEN RANGE

#include <stdio.h>

int isPrime(int num){
    for (int i = 2; i < num; i++){
        if(num%i == 0){
            return 0;
        }
    }
    return 1;
}

void main(){
    int n = 12, m = 40;
    for (int i = n; i <= m; i++) {
        if(isPrime(i)){
            printf("%d ", i);
        }
    }
}

File: practice/prime_between_range.py
# PRINT PRIME NUMBER IN A GIVEN RANGE

start = int(input("Enter first number : "))
end = int(input("Enter second number : "))

def isPrime(num):
    for i in range(2, num):
        if num % i == 0:
            return 0
    return 1

for i in range(start, end+1):
    if isPrime(i):
        print(i)
